other-guides/hosting-on-linux-server

```diff
@@ -0,0 +1,53 @@
+# Hosting on a Linux Server
+
+The best way to host your bot is on a Linux Server (or simply said: just a virtual server. Its also for bare metal servers).
+
+## Requirements
+
+- A Linux Server with a stable version, node, npm, git, pm2. 
+- Github repository from your bot (privat).
+- SSH Client
+
+## Getting Started
+
+### Login
+
+The best way to do that is with root access. If its root access the username would be `root`.
+The first step you do is to login into your Linux Server. You can use a SSH Client, with Windows CMD `ssh (username)@(ip)` or on macOS ssh 192.168.2.30 -l (username)`
```

Comment:
Not necessary for Windows, you can set up `%UserProfile%/.ssh/config` with something such as:

```yaml
Host my-server
  HostName 192.168.1.1
  Port 22
  User my-user
```

And then when you do `ssh my-server`, it automatically logs in.